Core/Src/torque.c

```diff
@@ -149,63 +202,53 @@ void vCalcTorque(void *pv_params)
 
 			func_state_t func_state = get_func_state();
 			if (func_state != ACTIVE) {
-				torque = 0;
+				dti_set_torque(0);
 				continue;
 			}
 
 			/* EV.4.7: If brakes are engaged and APPS signals more than 25% pedal travel, disable power
 			to the motor(s). Re-enable when accelerator has less than 5% pedal travel. */
 			//fault_data_t fault_data = { .id = BSPD_PREFAULT, .severity = DEFCON5 };
 			/* 600 is an arbitrary threshold to consider the brakes mechanically activated */
-			if (brake_value > 600 && (accelerator_value) > 0.25) {
-				printf("\n\n\n\rENTER MOTOR DISABLED\r\n\n\n");
-				motor_disabled = true;
-				torque = 0;
-			}
-
-			if (motor_disabled) {
-				printf("\nMotor disabled\n");
-				if (accelerator_value < 0.05) {
-					motor_disabled = false;
-					printf("\n\nMotor reenabled, queuing fault\n\n");
-					//queue_fault(&fault_data);
-				} else {
-					torque = 0;
-					dti_set_torque(torque);
-					continue;
-				}
-			}
-
-			drive_state_t drive_state =
-				AUTOCROSS; //get_drive_state();
+			// if (brake_value > 600 && (accelerator_value) > 0.25)
```

Comment:
we removing this for good?

Reply:
No cuz we need it for FSAE rules. We gotta adjust it to be better to the driver while also satisfying the judges. Kept it out and keeping it out for testing reasons. Gonna reintroduce it eventually.